### ISSUE TITLE
Feature request: Show VUPlayer in AutoPlay in Windows 10

### ISSUE THREAD

**Issue by clearwaterstream** [jfchapman/VUPlayer#8]
First, let me say this app is absolutely awesome. I needed something simple (software version of a hardware CD player) and this app fits the bill. The fact is supports Gracenote is really cool.

I am trying to make VUPlayer launch when an audio CD is inserted. When I got to Control Panel - All Items -> AutoPlay I don't see VUPlayer as an option under Audio CD section. Perhaps some registry entry needs to be inserted for it to be shown. I do see Foobar and PowerDVD 18.

Thanks!!

**Comment by jfchapman** [comment]
Thanks for trying out the app!

I took the decision to not include any registry handling code in version 4, and leave all that stuff in the hands of Windows itself.

Like you, I tried to work out how to add VUPlayer to the list of Audio CD handlers, and gave up. I can't remember if this was possible in Windows 7, but I certainly couldn't manage it in Windows 10. It seems to work differently compared to regular file type associations.

What I have done in version 4.9 is support Audio CD auto play on the command line. So, if you're feeling confident modifying the registry, it is possible to register VUPlayer as the handler.

I first registered Audio CD with Windows Media Player to see where the registry entry landed, and it was here:
`[HKEY_CLASSES_ROOT\AudioCD\shell\play\command]`
With default value:
`"%ProgramFiles(x86)%\Windows Media Player\wmplayer.exe" /prefetch:3 /device:AudioCD "%L"`

So, I backed up this entry, and changed the default value to:
`"%ProgramFiles%\VUPlayer 4\VUPlayer.exe" "%L"`

With this registry entry, when you insert an Audio CD, it will launch in VUPlayer and start playing.

I always feel a bit unclean when messing with the registry (see my first point), but it's a roundabout way of supporting auto play. Maybe one day I will figure out how to do this properly in windows 10.
